.github/workflows/lint.yml

```diff
@@ -33,7 +33,7 @@ jobs:
             go.sum
       - uses: golangci/golangci-lint-action@v8
         with:
-          version: latest
+          version: v2
```

Comment:
golangci-lint didn't run on this PR so I'm not 100% convinced this works. IMO we should delete the get diff step and always run golangci-lint b/c it doesn't take long.

https://github.com/celestiaorg/celestia-core/actions/runs/15759494414/job/44423931502?pr=2042

Reply:
Thanks! Let's see if golangci-lint passes now.